Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -59,6 +61,40 @@
             message="The file that you are downloading contains personal information. If you are on a public computer, please ensure that the file is deleted before you log off."
             @submit="generateMedicationHistoryPdf"
         />
+        <ProtectiveWordComponent
+            ref="protectiveWordModal"
+            :error="protectiveWordAttempts > 1"
+            :is-loading="isLoading"
+            @submit="onProtectiveWordSubmit"
+            @cancel="onProtectiveWordCancel"
+        />
+
+        <vue-html2pdf
+            ref="pdfGenerator"
+            :show-layout="false"
+            :enable-download="true"
+            :paginate-elements-by-height="950"
+            :html-to-pdf-options="{
+                margin: [15, 15],
+                filename: 'HealthGateway_MedicationHistory.pdf',
+                image: { type: 'jpeg', quality: 0.98 },
+                html2canvas: { scale: 2, letterRendering: true },
+                jsPDF: {
+                    unit: 'pt',
+                    format: 'letter',
+                    orientation: 'portrait',
+                },
+                pagebreak: { mode: ['avoid-all', 'css', 'legacy'] },
+            }"
+            pdf-format="a4"
```

Comment:
does this library support letter size (8.5 x 11) ?

Reply:
Yes